pyaedt/icepak.py

```diff
@@ -91,7 +91,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, ex_type, ex_value, ex_traceback):
-        ''' Push exit up to parent object Design '''
+        """Push exit up to parent object Design."""
```

Comment:
Thanks for fixing this.  It's annoying to have inconsistent standards for docstrings.